adapter-cqld4/src/main/java/io/nosqlbench/engine/extensions/vectormath/CqlUtilsPluginInfo.java

```diff
@@ -18,19 +18,19 @@
 
 import com.codahale.metrics.MetricRegistry;
 import io.nosqlbench.api.config.LabeledScenarioContext;
-import io.nosqlbench.api.extensions.ScriptingPluginInfo;
+import io.nosqlbench.api.extensions.ScriptingExtensionPluginInfo;
 import io.nosqlbench.nb.annotations.Service;
 import org.apache.logging.log4j.Logger;
 
-@Service(value = ScriptingPluginInfo.class,selector = "vectormath")
-public class VectorMathPluginInfo implements ScriptingPluginInfo<VectorMath> {
+@Service(value = ScriptingExtensionPluginInfo.class,selector = "cql_utils")
```

Comment:
Is the idea truly that this will in the future serve as a plugin for any cql utils or will it continue to be specific to vector stuff? Small quibble but it's named like the former but organized (package wise) like the latter.

Reply:
We should change the packaging, post merge to match "cql utilities". The idea is to decouple across the data structure while allowing them to be complimentary.